### PR TITLE
[CI] E2E Core workflow update

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
Ports the Cilium to the new core e2e workflow

This PR fixes https://github.com/meshery/meshery/issues/5529

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/e2etests.yaml [hunk 1]
```diff
@@ -41,10 +41,11 @@ jobs:
 
   TestCilium:
     needs: SetPatternfile
-    uses: meshery/meshery/.github/workflows/testadapters.yaml@master
+    uses: meshery/meshery/.github/workflows/test_adapters.yaml@master
     with:
-      expected_pods: cilium
```
Comment (gr455) [inline .github/workflows/e2etests.yaml:46]: I wonder why there is only 1 expected pod but 3 namespaces? // @MUzairS15

Comment (MUzairS15) [reply .github/workflows/e2etests.yaml:46]: @gr455 In actual cilium has 3 expected pods to come up  but due to some issue the other pods tend to get killed, so I removed from expected pods, but forgot to remove namespaces 😅.

I will address this by creating another PR.

Comment (gr455) [reply .github/workflows/e2etests.yaml:46]: Ah. Okay then

**Review by gr455 — APPROVED** [review]
@Revolyssup Ready to merge

**Comment by gr455** [comment]
> Signed-off-by: Gaurav Chadha [gauravchadha1676@gmail.com](mailto:gauravchadha1676@gmail.com)
> 
> **Description** Ports the Cilium to the new core e2e workflow
> 
> This PR fixes [meshery/meshery#5529](https://github.com/meshery/meshery/issues/5529)
> 
> **Notes for Reviewers**
> 
> **[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
> 
> * [x]  Yes, I signed my commits.

@Chadha93 Can you just change 'This PR fixes' to 'This PR is a part of' so that it doesn't close the issue on merge

**Review by Revolyssup — APPROVED** [review]

**Pull request merged**